packages/api/src/utils/login.ts

```diff
@@ -60,6 +60,23 @@ export interface AzureAccountProvider {
      * @param subscriptionId user used subscription id
      */
     setSubscription(subscriptionId: string): Promise<void>;
+
+    /**
+     * Get account information
+     */
+    getAccountInfo(): Record<string, string> | undefined;
+
+    /**
+     * Get user select subscription, tenant information
+     * subscriptionId, tenantId...
+     */
+    getSelectedSubscription(): Record<string, string> | undefined;
```

Comment:
Make this an interface?

Reply:
Reuse SubscriptionInfo to be return type.